environment/src/main/java/com/sequenceiq/environment/experience/common/CommonExperience.java

```diff
@@ -1,28 +1,41 @@
 package com.sequenceiq.environment.experience.common;
 
-import java.util.StringJoiner;
+import static org.slf4j.LoggerFactory.getLogger;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.apache.commons.lang3.StringUtils;
 
 public class CommonExperience {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(CommonExperience.class);
-
     private String name;
 
     private String description;
 
+    private String businessName;
+
     private String internalEnvironmentEndpoint;
 
+    private String baseAddress;
+
+    private String policyEndpoint;
+
+    private String policyPort;
+
     private String address;
 
-    public CommonExperience(String name, String description, String internalEnvironmentEndpoint, String address) {
+    private String environmentEndpointPort;
+
+    public CommonExperience(String name, String description, String internalEnvironmentEndpoint, String address, String businessName, String policyEndpoint,
```

Comment:
There are too much constructor parameters, can we have a builder for this class instead of adding 5 additional parameters?

Reply:
This common experience is a config file related class that will be read from the application yaml, therefore - outside the tests - the creation of these instances is up to the spring